src/main/java/fr/openmc/core/utils/text/TextManager.java

```diff
@@ -0,0 +1,132 @@
+package fr.openmc.core.utils.text;
+
+import org.bukkit.configuration.file.YamlConfiguration;
+import org.bukkit.configuration.file.FileConfiguration;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import fr.openmc.core.OMCPlugin;
+
+
+public class TextManager {
+
+    private final String defaultLanguage;
+    private final OMCPlugin plugin;
+    private final File translationFolder;
+    private final Map<String, FileConfiguration> loadedLanguages;
+
+    public TextManager(OMCPlugin plugin, File translationFolder, String defaultLanguage) {
```

Comment:
Met loadLanguage() dans ton TextManager ce qui fait que tu peux enlever la variable dans le OMCPlugin et ça textManager.loadAllLanguages();
Tu devrais juste avoir ça dans ton OMCPlugin dcp
new TextManager(this, new File(this.getDataFolder(), "translations"), "fr");

Reply:
En faisant cela, tu perds en dynamisme. Par contre, si à l'avenir il doit être possible de modifier ce langage depuis une commande, c'est beaucoup plus simple de faire un appel à une méthode publique `loadLanguage()` que de devoir refaire une instance de `TextManager`.
Par contre, si j'ai bien compris le système, le nom de `TextManager` n'est pas forcément très approprié. Peut-être trouver un meilleur nom serait une bonne idée.